Simbody/include/simbody/internal/Force_Custom.h

```diff
@@ -247,6 +247,13 @@ class SimTK_SIMBODY_EXPORT Force::Custom::Implementation {
     virtual bool dependsOnlyOnPositions() const {
         return false;
     }
+    /**
+     * Called during the realizeSubsystemDynamicsImpl method in GenralForceSubsystemRep, used to determine whether a custom force
+     * should be parallelized. By default, this method returns false and can be overwritten by the force subclass.
+     */
+    virtual bool isParallelByDefault() const {
```

Comment:
I think that this change should probably be implemented in Simbody too, correct? It would make sense for the method names to be the same.

Relevant Issue: https://github.com/simbody/simbody/pull/414#discussion_r35394639